src/Jackett.Common/Definitions/matrix.yml

```diff
@@ -0,0 +1,184 @@
+    movie-search: [q, imdbid]
+    book-search: [q]
+    music-search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+  - name: freeleech
+    type: checkbox
+    label: Filter freeleech only
+    default: false
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: 0
+    options:
+      0: created
+      1: title
+      4: size
+      6: seeders
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+  - name: info_tpp
+    type: info
+    label: Results Per Page
+    default: For best results, change the <b>Torrents per page:</b> setting to <b>100</b> on your account profile.
+  - name: info_activity
+    type: info
+    label: Account Inactivity
+    default: "If you do not log in to the site for 6 months, the system will automatically delete it!"
+
+login:
+  path: takelogin.php
+  method: post
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+    logout: ""
+  error:
+    - selector: td.embedded:contains("Sikertelen bejelentkezés")
+  test:
+    path: /
+    selector: a[href$="logout.php"]
+
+search:
+  paths:
+    - path: browse.php
+  inputs:
+    $raw: "{{ range .Categories }}c{{.}}=1&{{end}}"
+    search: "{{ if .Query.IMDBID }}{{ .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}"
+    # 0 active only, 1 include dead, 2 only dead
+    incldead: 1
+    sort: "{{ .Config.sort }}"
+    type: "{{ .Config.type }}"
+
+  rows:
+    selector: "table.mainouter > tbody > tr > td > table tr:has(a[href^=\"download.php?id=\"]){{ if .Config.freeleech }}:has(img[src$=\"/pic/ingyentorrent.gif\"]){{ else }}{{ end }}"
+
+  fields:
+    category:
+      selector: a[href^="browse.php?cat="]
+      attribute: href
+      filters:
+        - name: querystring
+          args: cat
+    title:
+      selector: a.torrent-link b
+    _id:
+      selector: a[href^="download.php?id="]
+      attribute: href
+      filters:
+        - name: querystring
+          args: id
+    details:
+      text: "details.php?id={{ .Result._id }}"
+    download:
+      selector: a[href^="download.php?id="]
+      attribute: href
+    poster:
+      selector: a.torrent-link
+      attribute: data-cover
+    imdbid:
+      selector: a[href*="imdb.com/title/tt"]
+      attribute: href
+    files:
+      selector: td:nth-child(5) a
+    date:
+      selector: td:contains("Feltöltve:")
+      filters:
+        - name: regexp
+          args: "Feltöltve: *([0-9]{4}-[0-9]{2}-[0-9]{2} +[0-9]{2}:[0-9]{2}:[0-9]{2})"
+        - name: append
+          args: " +01:00" # Europe/Budapest, CET
+        - name: dateparse
+          args: "yyyy-MM-dd HH:mm:ss zzz"
+    grabs:
+      selector: td:nth-child(7) b
+    size:
+      selector: td:nth-child(7)
+      filters:
+        - name: regexp
+          args: "([0-9]+\\.?[0-9]* *[kKmMgGtTpP]i?[bB])"
```

Comment:
```suggestion
          args: "(.+?B)"
```

Reply:
Same point as above for extracting the date/time stamp. More expressive and less likely to mismatch in confusing ways.

Reply:
If there is such an extreme change that letters other than `B` are included before the end of size, we'd need to take a look at the site anyway.

Reply:
> If there is such an extreme change that letters other than `B` are included before the end of size, we'd need to take a look at the site anyway.

In some cases maybe, but in other cases, maybe not and it need not be extreme. This column already has two fields in the same table cell. If the order or format changes in the future, then the more explicit regex is more likely to still work.

Regardless, debugability also wins in this case. Agreed.